pkg/controllers/metrics.go

```diff
@@ -9,17 +9,6 @@ import (
 )
 
 var (
-	// same as requestLatency in
-	// "sigs.k8s.io/controller-runtime/pkg/metrics"
-	// its not exported, yet we want to remove it
-	// so we copy it here
-	requestLatencyCollector = prometheus.NewHistogramVec(prometheus.HistogramOpts{
-		Subsystem: metrics.RestClientSubsystem,
-		Name:      metrics.LatencyKey,
-		Help:      "Request latency in seconds. Broken down by verb and URL.",
-		Buckets:   prometheus.ExponentialBuckets(0.001, 2, 10),
-	}, []string{"verb", "url"})
```

Comment:
I think this was actually failing because the upstream metric used `[]string{"verb", "host"}`, so this Unregister didn't work.

Reply:
It works in v0.14.6, see
https://github.com/kubernetes-sigs/controller-runtime/blob/cd65cb25d314f40a329a688f4714fe3282589e97/pkg/metrics/client_go_adapter.go#L58-L63

Reply:
They've changed it since v0.14.7

Reply:
Let's check why and how they changed it before we make decisions

Reply:
Oh, looks like they decided to remove these metrics just as we did. 
https://github.com/kubernetes-sigs/controller-runtime/commit/139e0d728e115fdeb4718fb2de18eacb9fd85f91

So it's okey to remove these lines in our code.